online-inference/fastertransformer/client/example.py

```diff
@@ -79,20 +91,21 @@ def prepare_tensor(client, name, input):
 
 def main(config, request):
     client_type = httpclient if config['protocol'] == 'http' else grpcclient
-    with client_type.InferenceServerClient(config['url'], verbose=config['verbose'], concurrency=10) as cl:
+    with client_type.InferenceServerClient(config['url'], verbose=config['verbose'], concurrency=1, connection_timeout=1000.0, network_timeout=1000.0,) as cl:
```

Comment:
Why such long timeout? If that is in seconds that will make users wait a lot to get an error

Reply:
Will change to 100 seconds.